src/taxonomy/taxonomy-menu/TaxonomyMenu.test.jsx

```diff
@@ -88,29 +84,34 @@ describe.each([true, false])('<TaxonomyMenu iconMenu=%s />', async (iconMenu) =>
   });
 
   test('should open and close menu on button click', async () => {
```

Comment:
As we removed the `findByTestId` await, this could go.
```suggestion
  test('should open and close menu on button click', () => {
```